src/ontology/hp-edit.owl

```diff
@@ -140931,6 +141010,555 @@ AnnotationAssertion(dcterms:date <http://purl.obolibrary.org/obo/HP_0020227> "20
+AnnotationAssertion(dcterms:date <http://purl.obolibrary.org/obo/HP_0020257> "2023-11-10T19:32:09Z"^^xsd:dateTime)
+AnnotationAssertion(rdfs:comment <http://purl.obolibrary.org/obo/HP_0020257> "This pattern may be observed in mantle cell lymphoma.")
+AnnotationAssertion(rdfs:label <http://purl.obolibrary.org/obo/HP_0020257> "Mantle zone pattern")
+SubClassOf(<http://purl.obolibrary.org/obo/HP_0020257> <http://purl.obolibrary.org/obo/HP_0020251>)
+
+# Class: <http://purl.obolibrary.org/obo/HP_0020258> (Germinal centers with absent marginal zones)
+
+AnnotationAssertion(dcterms:creator <http://purl.obolibrary.org/obo/HP_0020258> <https://orcid.org/0000-0002-0736-9199>)
+AnnotationAssertion(dcterms:date <http://purl.obolibrary.org/obo/HP_0020258> "2023-11-10T19:32:58Z"^^xsd:dateTime)
+AnnotationAssertion(rdfs:label <http://purl.obolibrary.org/obo/HP_0020258> "Germinal centers with absent marginal zones")
+SubClassOf(<http://purl.obolibrary.org/obo/HP_0020258> <http://purl.obolibrary.org/obo/HP_0020250>)
+
+# Class: <http://purl.obolibrary.org/obo/HP_0020261> (Poorly defined primary lymph-node follicles)
+
+AnnotationAssertion(<http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/HP_0020261> "Lack of the typical easily recognized delineation of primary follicles within lymph nodes.")
+AnnotationAssertion(dcterms:creator <http://purl.obolibrary.org/obo/HP_0020261> <https://orcid.org/0000-0002-0736-9199>)
+AnnotationAssertion(dcterms:date <http://purl.obolibrary.org/obo/HP_0020261> "2023-11-10T19:36:39Z"^^xsd:dateTime)
+AnnotationAssertion(rdfs:comment <http://purl.obolibrary.org/obo/HP_0020261> "Follicles generally are relatively distinct at low magnification on H&E stained slides, however may not be obvious on the H&E stained slide, requiring immunohistochemistry to confirm the presence of follicle formation.")
+AnnotationAssertion(rdfs:label <http://purl.obolibrary.org/obo/HP_0020261> "Poorly defined primary lymph-node follicles")
+SubClassOf(<http://purl.obolibrary.org/obo/HP_0020261> <http://purl.obolibrary.org/obo/HP_0020242>)
+
+# Class: <http://purl.obolibrary.org/obo/HP_0020262> (Progressive transformation of germinal centers)
+
+AnnotationAssertion(Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> "PMID:14502795") <http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/HP_0020262> "This is a condition sometimes seen in a subset of germinal centers of reactive lymph nodes where germinal centers become enlarged and infiltrated by benign mantle zone cells and some T cells.")
+AnnotationAssertion(dcterms:creator <http://purl.obolibrary.org/obo/HP_0020262> <https://orcid.org/0000-0002-0736-9199>)
+AnnotationAssertion(dcterms:date <http://purl.obolibrary.org/obo/HP_0020262> "2023-11-10T19:37:12Z"^^xsd:dateTime)
+AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#hasExactSynonym> <http://purl.obolibrary.org/obo/HP_0020262> "PTGC")
```

Comment:
Missing abbreviation type?

Reply:
The sysnnym type seems tpo be broken in that it is giving errors but let's add it!

Reply:
I will do this in a seperate, general PR.